packages/core/src/app/order/OrderSummaryPrice.tsx

```diff
@@ -146,4 +153,12 @@ class OrderSummaryPrice extends Component<OrderSummaryPriceProps, OrderSummaryPr
     };
 }
 
-export default OrderSummaryPrice;
+function mapToOrderSummaryPriceProps({ checkoutState }: CheckoutContextProps): WithCheckoutOrderSummaryPriceProps {
+    const { statuses: { isSubmittingOrder } } = checkoutState;
+
+    return {
+        isSubmittingOrder: isSubmittingOrder(),
+    };
+}
+
+export default withCheckout(mapToOrderSummaryPriceProps)(OrderSummaryPrice);
```

Comment:
Could we avoid using HOC and creating new enzyme tests?
Because we are going to upgrade to react 18 next year, all these work will be outdated at that time.